js/src/edit-block/components/field.js

```diff
@@ -93,7 +93,7 @@ const Field = ( {
 		<div
 			role="row"
 			className={ className(
-				'field w-full',
+				'relative w-full rounded-sm border border-gray-400 hover:border-black',
```

Comment:
This should fix the issue where focusing and hovering had a strange mix of blue and black.

Now, on focusing and hovering on a field, it's just blue:
![the-hovering](https://user-images.githubusercontent.com/4063887/105559674-63fa3c80-5cd7-11eb-8411-617871e77035.gif)